app/controllers/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email:  params[:email], password: params[:password])
```

Comment:
Kindly remove commented code, you can always use git history to access previous code right :slightly_smiling_face: